src/AwsCrypto.dfy

```diff
@@ -8,19 +8,19 @@ module Aws {
   datatype Outcome =
     | AWS_OP_SUCCESS
     | AWS_OP_ERR
-    /** The ciphertext was malformed or corrupt */
+      /** The ciphertext was malformed or corrupt */
     | AWS_CRYPTOSDK_ERR_BAD_CIPHERTEXT /* 0x2000 */
     /** An unknown internal error has occurred */
```

Comment:
Why not this on?

Reply:
I found a bug related to parsing nested comments in the formatter and I fixed it:
https://github.com/dafny-lang/dafny/pull/2399/commits/b45b2d85c43525fdad583bf38d244ae65a02a843